tests/test_model_backward_compatibility.py

```diff
@@ -15,7 +15,7 @@ def set_seed(seed):
     random.seed(seed)
 
 
-@pytest.mark.parametrize("model_fn", [pretrained.orb_v1, pretrained.orb_v2])
+@pytest.mark.parametrize("model_fn", [pretrained.orb_v2])
```

Comment:
I liked the numerically exact tests before as I do think there is a small class of bugs it catches that less precise tests do not (e.g. we did recently introduce small amounts of stochasticity into our internal models at inference time by mistake)

But also, it's not a hill I would die on, so your call.

Reply:
Exact tests are good, but create a big maintenance burden. It also means that we may miss changes between models which indicate bugs, as this test would have to be updated for each model release - let's try this for now and if we find it is not precise enough we can switch back.